Framework/PythonInterface/mantid/plots/datafunctions.py

```diff
@@ -1101,7 +1101,7 @@ def add_colorbar_label(colorbar, axes):
     :param axes: the axes that the colorbar belongs to.
     """
     colorbar_labels = [ax.colorbar_label for ax in axes if hasattr(ax, 'colorbar_label')]
-    if colorbar_labels.count(colorbar_labels[0]) == len(colorbar_labels):
+    if colorbar_labels and colorbar_labels.count(colorbar_labels[0]) == len(colorbar_labels):
```

Comment:
Is this intentional?

Reply:
It is. For some reason colorbar_labels is sometimes empty, and it then throws an error. Checking before accessing the first element prevent that (and looks like a good policy, moreover).

Reply:
Would be nice to understand why it is empty, but that's beyond this PR.